curator-framework/src/main/java/org/apache/curator/framework/api/CreateBuilder.java

```diff
@@ -18,8 +18,19 @@
  */
 package org.apache.curator.framework.api;
 
-public interface CreateBuilder extends
-    CreateBuilderMain
+public interface CreateBuilder extends CreateBuilderMain
```

Comment:
Right. That's not a problem.

`public interface CreateBuilder extends CreateBuilder2 { CreateBuilder2 orSetData() }`